tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -1639,4 +1639,22 @@ def test_trim_flow_name(self):
         long_stripped, _ = re.subn(r'\s', '', long)
         self.assertEqual(short, SklearnExtension.trim_flow_name(long_stripped))
 
-        self.assertRaises(ValueError, SklearnExtension.trim_flow_name, "weka.IsolationForest")
+        long = """sklearn.pipeline.Pipeline(
```

Comment:
Could you please add unit tests for feature union and simple non-pipeline, non-feature selection models here, too?

Reply:
Good catch. FeatureUnion did not work as expected. Added the tests and fixed the issue.